.github/stale.yml

```diff
@@ -3,7 +3,8 @@
 # Number of days of inactivity before an Issue or Pull Request becomes stale
 daysUntilStale: 60
 # Number of days of inactivity before a stale Issue or Pull Request is closed
-# daysUntilClose: 7
+# 2**32-1 until https://github.com/probot/stale/issues/79 is fixed
+daysUntilClose: 4294967295
```

Comment:
Heads up that this is causing an error and preventing stale from running against this repository: `RangeError: errorInvalid time value`

Reply:
Thanks for reporting. What's the biggest number we can put here?

Reply:
Just saw you suggested a number in the original post.